certification/internal/shell/scorecard_olm_suite.go

```diff
@@ -0,0 +1,134 @@
+package shell
```

Comment:
The same comments as above for this Check :)

Reply:
Gotcha.